README.md

```diff
@@ -399,9 +399,12 @@ ARGUMENTS
   ACTORNAME  Name of the actor and its directory
 
 OPTIONS
-  -t, --template=template  Template for the actor. If not provided, the command will prompt for it.Visit
-                           https://raw.githubusercontent.com/apify/actor-templates/master/templates/manifest.json to
-                           find available template names.
+  -t, --template=template      Template for the actor. If not provided, the command will prompt for it.
+                               Visit
+                               https://raw.githubusercontent.com/apify/actor-templates/master/templates/manifest.json to
+                               find available template names.
+
+  --dont-install-dependencies  Skip installing actor dependencies.
```

Comment:
What about `--skip-dependency-install` or sth? I really don't like the word 'dont` in CLI argument :D

Reply:
Yeah, I like that better 😄